webapp/app.py

```diff
@@ -76,11 +75,13 @@ def results():
 @app.route("/careers/tech-ops", methods=["GET", "POST"])
 def department_group():
     department = flask.request.path.split("/")[2]
-    vacancies = get_vacancies(department)
+    vacancies = greenhouse_api.get_vacancies(department)
 
     if flask.request.method == "POST":
-        response = submit_to_greenhouse(
-            flask.request.form, flask.request.files
+        response = greenhouse_api.submit_application(
+            os.environ["GREENHOUSE_API_KEY"],
```

Comment:
Can this go into a varaible that gets instantiate on start of the app.

Reply:
No because we don't want it to fail until it's needed. As in, you can run the site fine without this API key until you try to submit an application, then only it fails.

Reply:
Oh alright I'll make it prettier